src/store/task/reducers/taskReducers.js

```diff
@@ -0,0 +1,21 @@
+import { taskActionTypes } from "src/store/task/types/taskTypes";
+
+const { ADD_TASK } = taskActionTypes;
```

Comment:
@ray-wtag vai, `taskActionTypes` is an object and `ADD_TASK` (later `EDIT_TASK` and so on...) are properties of `taskActionTypes`.

That's why `ADD_TASK` couldn't be imported directly in the first line.

I was wondering is it a bad idea to group those in an object instead of declaring separate const and exporting them individually for each task type?

Reply:
I guess exporting separately will help to destructure when you are importing, you don't have to destructure in a new line here.

Reply:
@ray-wtag done vai.